packages/neovim/src/utils/findNvim.ts

```diff
@@ -31,6 +31,10 @@ export type FindNvimOptions = {
    *   - Example: `['0.4.4', '0.5.0', '0.4.3']`
    */
   readonly orderBy?: 'desc' | 'none';
+  /**
+   * (Optional) Stop searching after found a valid match
+   */
+  readonly stopOnFirstMatch?: boolean;
```

Comment:
Can use a shorter name, the docstring clarifies any questions.

Reply:
```suggestion
  readonly firstMatch?: boolean;
```

Reply:
Fixed